components/org.wso2.carbon.identity.data.publisher.authentication.audit/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/audit/AuthenticationAuditLoggingHandler.java

```diff
@@ -179,18 +182,18 @@ protected void doPublishAuthenticationFailure(AuthenticationAuditData authentica
 
         AUDIT_LOG.info(String.format(
                 FrameworkConstants.AUDIT_MESSAGE,
-                authenticationData.getAuthenticatedUser(),
+                LoggerUtils.maskContent(authenticationData.getAuthenticatedUser()),
                 "Login",
                 "ApplicationAuthenticationFramework", auditData, FrameworkConstants.AUDIT_FAILED));
     }
 
-    protected void doPublishSessionTermination(AuthenticationContext context, String username,
+    protected void doPublishSessionTermination(AuthenticationContext context, String username, String userId,
```

Comment:
shouldn't change the method signature of protected methods

Reply:
fixed